docs/static/breaking-changes.asciidoc

```diff
@@ -1,5 +1,16 @@
 [[breaking-changes]]
-== Breaking Changes
+
+== Breaking Changes Across PQ Versions
+
+If you are upgrading between Logstash versions and use the persistent queue, we strongly recommend draining or deleting
+your persistent queue prior to the upgrade due to data incompatibilities. We are currently working on resolving these issues in our 6.3.0 release, but extra steps will need to be taken until then.
+
+Drain the persistent queue be starting Logstash with the `queue.drain` setting enabled, followed by signalling Logstash to shutdown, then waiting for it to shutdown cleanly. This may take some time if you have a large queue backlog.
```

Comment:
This seems accurate, if a little overly-complex in practice.

It would be nice to have an option that doesn't start up any of a pipeine's inputs, but otherwise uses the pipeline to drain the queue.

Reply:
+1 . Would you bind opening a new issue to track that? I think that makes a lot of sense.